.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+aliases:
+  - &restore-cache
+    keys:
+      - dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+      # Fallback in case checksum fails
+      - dependencies-{{ .Branch }}-
+
+  - &save-cache
+    paths:
+      - node_modules
+      - website/node_modules
+    key: dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+
+  - &yarn-install
+    run: |
+      sudo npm i -g yarn@^0.27.5
+      yarn --no-progress
+
+  - &yarn-install-no-sudo
+    run: |
+      npm i -g yarn@^0.27.5
+      yarn --no-progress
+
+  - &deploy
+    command: |
+      # Deploy Jest website
+      git config --global user.email "jest-bot@users.noreply.github.com"
+      git config --global user.name "Website Deployment Script"
+      echo "machine github.com login jest-bot password $GITHUB_TOKEN" > ~/.netrc
+      # crowdin install start
+      sudo apt-get install default-jre
+      wget https://artifacts.crowdin.com/repo/deb/crowdin.deb -O crowdin.deb
+      sudo dpkg -i crowdin.deb
```

Comment:
Ideally all required software should be installed in a Docker image so that you don't incur the cost every single time the build runs. That's one of the main advantages of CircleCI 2.0 :)

Basically, there should be some Jest dev Docker image that extends whichever other Docker images you're using, and has this stuff installed.